client/src/Components/DashboardPage.jsx

```diff
@@ -289,5 +289,6 @@ DashboardPage.propTypes = {
   accountData: PropTypes.object,
   handleAddTransaction: PropTypes.func,
   loading: PropTypes.bool.isRequired,
+  updateAccountData: PropTypes.func,
```

Comment:
aside from this, this was just the linter going wild on dashboardPage--shouldn't be any other actual changes